### PR TITLE
Change old variables to new proxy variables

### PR DESCRIPTION
**Pull request by nicklathe**
- Feed etcd quion new proxy variables

### CODE REVIEW

**Comment by dragan** [comment]
We're going to create an integration branch and break this PR apart to keep track of the changes at a smaller scope.

File: kubernetes/main.tf [hunk 1]
```diff
@@ -151,6 +151,26 @@ variable "kubernetes_version" {
...
+  default     = "gcr.io/google_containers/pause-amd64"
+}
+
+variable "pod_infra_version" {
+  description = "Version of pod infra"
+  default     = "3.0"
+}
+
+variable "flannel_image_repo" {
+  description = "Docker image repository for flannel image"
```
Comment (weitenghuang) [inline kubernetes/main.tf]: Is flannel rkt image?

Comment (dragan) [reply kubernetes/main.tf]: Based on the wrapper script yes, but rocket can run docker images as well. We have ways to specify that it's docker.

Comment (weitenghuang) [reply kubernetes/main.tf]: do you mean we will disable default flanneld.service, and create custom flanneld?

Comment (dragan) [reply kubernetes/main.tf]: Nope, using the same image, no customization, just where we pull it from. Also, we will be closing this PR. Comments should be on #51.

**Comment by dragan** [comment]
Closing as we have broken this PR down into smaller changes.

**Pull request closed without merging**